spark/src/main/java/org/apache/zeppelin/spark/SparkSqlInterpreter.java

```diff
@@ -129,8 +116,15 @@ public InterpreterResult interpret(String st, InterpreterContext context) {
       sc.setLocalProperty("spark.scheduler.pool", null);
     }
 
+    Object rdd = null;
+    try {
+      Method sqlMethod = sqlc.getClass().getMethod("sql", String.class);
+      rdd = sqlMethod.invoke(sqlc, st);
```

Comment:
It's because of method signature is not exactly same. 
version 1.3 and later has `def sql(sqlText: String): DataFrame` while version 1.2 and prior has `def sql(sqlText: String): SchemaRDD`